experiments/AtmosGCM/moist-baroclinic-wave-bulksfcflux.jl

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/env julia --project
```

Comment:
Not sure if you want to add this experiment here since we'll be replacing it right away? But adding it is fine too.